<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TissueClassificationDialog</class>
 <widget class="QDialog" name="TissueClassificationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tissue Classification</string>
  </property>
  <widget class="QPushButton" name="runStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Step &gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spatialPriorWeightingEditBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spatial Prior</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tissue Classification (PVC)</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>tissue classification</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextTool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stage: Cerebrum Extraction</string>
   </property>
  </widget>
  <widget class="QLabel" name="autosaveLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreFromPreviousSessionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore From Previous...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
